### ISSUE TITLE
[T8][Accessibility][Functional]Unable to open and select the filters under Assessment column in Social History tab

### ISSUE THREAD

**Issue by nehabug** [Mohanme13/Feb-19-Bug-bash#369]
Unable to open and select the filters under Assessment column in Social History tab
